### DISCUSSION TITLE
Out of Memory

### DISCUSSION THREAD

**Discussion by baiyibing123** [jertel/elastalert2#1294] [Q&A] [answered]
<img width="1304" alt="image" src="https://github.com/jertel/elastalert2/assets/14048464/792f2b83-a7b4-4286-982d-7c4e97e31555">

不清楚发生了什么，一直这样

**Comment by jertel** [comment, marked as answer]
See #11 and provide the requested information. Additionally, include any recent changes from before the problem started.
